### PR TITLE
BUG use proper json for versions and always update dockerfile base env

### PR DESCRIPTION
**Pull request by beckermr**
Checklist
* [x] Pushed the branch to main repo
* [x] CI passed on the branch

### CODE REVIEW

File: conda_forge_webservices/update_me.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 import shutil
 import logging
 import github
+import json
```
Comment (beckermr) [inline conda_forge_webservices/update_me.py:8]:
```suggestion
```

File: conda_forge_webservices/update_me.py [hunk 2]
```diff
@@ -23,14 +23,12 @@ def _run_git_command(args):
 
 
 def get_current_versions():
-    r = subprocess.run(["conda", "list"], capture_output=True)
-    out = r.stdout.decode("utf-8")
+    r = subprocess.run(["conda", "list", "--json"], capture_output=True)
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:26]: Should we raise any process errors (in the unlikely event they occur)?

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True)
```

Comment (beckermr) [reply conda_forge_webservices/update_me.py:26]: sure? The server will 500 in this case and kill the gha job.

File: conda_forge_webservices/update_me.py [hunk 3]
```diff
@@ -23,14 +23,12 @@ def _run_git_command(args):
 
 
 def get_current_versions():
-    r = subprocess.run(["conda", "list"], capture_output=True)
-    out = r.stdout.decode("utf-8")
+    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True)
+    out = json.loads(r.stdout.decode("utf-8"))
```
Comment (jakirkham) [inline conda_forge_webservices/update_me.py:27]: Also in trying things locally, it looks like this saves us the `decode` step

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True, encoding="utf-8")
    out = json.loads(r.stdout)
```

Comment (jakirkham) [reply conda_forge_webservices/update_me.py:27]: Another option might be

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True, text=True)
    out = json.loads(r.stdout)
```

File: conda_forge_webservices/update_me.py [hunk 4]
```diff
@@ -23,14 +23,12 @@ def _run_git_command(args):
 
 
 def get_current_versions():
-    r = subprocess.run(["conda", "list"], capture_output=True)
-    out = r.stdout.decode("utf-8")
+    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True, encoding="utf-8")
```
Comment (beckermr) [inline conda_forge_webservices/update_me.py:26]:
```suggestion
    r = subprocess.run(
        ["conda", "list", "--json"],
        capture_output=True,
        check=True,
        encoding="utf-8",
    )
```

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks Matt! 🙏

Merge when passing

**Pull request merged**